### ISSUE TITLE
Hide Mod List?

### ISSUE THREAD

**Issue by MaesterEnzo** [steven4547466/LC-API#31]
Is there a way to hide the yellow pop-up at the start of a match that asks other players for installed mods?

Mod List
Asking all players for installed mods.

**Comment by steven4547466** [comment]
Not at the moment.

**Comment by MaesterEnzo** [comment]
It also always says this when dropping new loot into the ship.

Got Scrap
To sell, route to the company building

**Comment by NeuTraLZero** [comment]
Can confirm that the HideModList mod don't work anymore since LC API 3.4.0

And also the tip messages in the yellow box that's displayed for things like collecting scraps and routing to the company keep reappearing when meeting the conditions for the message

**Comment by steven4547466** [comment]
The way the tip is displayed and sent has changed, so if there was a mod that disabled it, that would have broken.

> And also the tip messages in the yellow box that's displayed for things like collecting scraps and routing to the company keep reappearing when meeting the conditions for the message

Could you show me a screenshot of what you mean so I can recreate the issue?

**Comment by steven4547466** [comment]
Actually I see the issue.

**Comment by steven4547466** [comment]
Alright the intro tips playing should be fixed in 3.4.3

**Comment by Archer-Dante** [comment]
Agreed, please add to the config an option to disable those popups!
